app/assets/javascripts/taxs.js

```diff
@@ -66,5 +66,7 @@ $(function() {
 
     $('.error-summary').focus();
 
+    document.getElementById('proposition-menu').querySelector('.js-header-toggle.menu').setAttribute("aria-hidden", "true");
```

Comment:
is there a reason you're just not setting `aria-hidden="true"` in the html or is this toggled at some point?

Reply:
Yes html is in TaaS. I thought about adding it there but then for other services it would also hide it where the css is not there